### PR TITLE
Task#7498 - branch was created from #7494. 

### PR DESCRIPTION
**Pull request by HeorhiiBoliuk**
Styled the sidebar, the body, prev/next/ user blocks in article page.
https://redmine.devbranch.work/issues/7498

### CODE REVIEW

**Comment by HeorhiiBoliuk** [comment]
Do edits at the views in node and front page

File: web/themes/custom/my_awesome_theme/templates/views/views-view--recent-articles.html.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div  class="recent-posts">
+  <h4>
+    <span>
+      RECENT POSTS
```
Comment (KuzyaWkk) [inline web/themes/custom/my_awesome_theme/templates/views/views-view--recent-articles.html.twig:4]: Need to use translatable rows. You can use a trans block or |t.
E.G. Recent Posts |t

**Review by KuzyaWkk — CHANGES_REQUESTED** [review]
Add translatable rows

**Review by KuzyaWkk — APPROVED** [review]

File: web/themes/custom/my_awesome_theme/templates/views/views-view--recent-articles.html.twig [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div  class="recent-posts">
+  <h4>
+    <span>
+      {% trans %}
+        RECENT POSTS
+      {% endtrans %}
```
Comment (lavrenchuk-dev) [inline web/themes/custom/my_awesome_theme/templates/views/views-view--recent-articles.html.twig:6]: It doesn't matter much but for the future
For simple string you can use {{ 'Recent Posts'|t }}
It's called twig filter

A trans block is generally for more complicated things.

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**